content/docs/rfcs/28/README.md

```diff
@@ -0,0 +1,56 @@
+---
+slug: 28
+title: 28/STATUS-FEATURING
+name: Status community featuring using waku v2
+status: raw
+tags: waku-application
+editor: Szymon Szlachtowicz <szymon.s@ethworks.io>
+---
+
+This spec is a proposition of voting on community featuring over waku v2.
+
+# Overview
+
+When there is a active community that is seeking new members, current users of community should be able to feature their community so that it will be accessible to larger audience.
+Status community curation DApp should provide such a tool.
+
+Rules of featuring:
+    - Given community can't be featured twice in a row.
+    - Only one vote per user per community (single user can vote on multiple communities)
+    - Voting will be done off-chain
+    - If community hasn't been featured votes for given community are still valid for the next 4 weeks
+
+Since voting for featuring is similar to polling solutions proposed in this spec could be also used for different applications.
+
+# Voting
+
+Voting for featuring will be done through waku v2. 
+
+Payload of waku message will be :
+```ts
+type FeatureVote = {
+    voter: string // address of a voter
+    sntAmount: BigNumber // amount of snt voted on featuring
```

Comment:
I though the votes were not weighted:

> Only one vote per user per community (single user can vote on multiple communities)

Reply:
One vote per user per community meaning that only one of waku votes will be counted towards community featuring but this vote can have multiple SNT which determines the weight of the vote just as in community addition and removal from directory. 

Although we can also simply check if user has more SNT than threshold and count this as one vote.